qemu/tests/cfg/virtio_driver_sign_check.cfg

```diff
@@ -1,16 +1,62 @@
+    only Windows
     type = virtio_driver_sign_check
-    cdroms += " sdk virtio"
-    cdrom_virtio = isos/windows/virtio-win.iso
-    cdrom_sdk = isos/windows/GRMSDKX_EN_DVD.iso
+    cdroms += " virtio"
+    cdrom_virtio = isos/windows/virtio-win.latest_prewhql.iso
+    floppy_readonly = yes
     floppies = "fl"
-    floppy_name = images/win7-64/virtio-win.vfd
-    list_files_cmd = dir /s /b %s | find "%s"
-    winsdk_au3 = autoit/winsdk.au3
-    signtool_install = "D:\autoit3.exe c:\winsdk.au3"
-    signtool_cmd = "c:\Program Files\Microsoft SDKs\Windows\v7.0\Bin\signtool.exe" verify /v /kp /c %s %s
-    drive_list = F: A:
+    list_files_cmd = 'dir /s /b %s | find "%s"'
+    get_disk_letter_cmd = 'wmic logicaldisk | find "virtio-win"'
+    drive_volume = A:
+    tested_drivers = Balloon pvpanic viorng vioserial vioinput NetKVM netkvm vioscsi viostor   
+
+    i386:
+        floppy_name = isos/windows/virtio-win.latest_prewhql.vfd.i386
+        arch_dir = 86
+        Win2008..sp2:
+            signtool = "WIN_UTILS:\signtool\signtool-WLK\Signtool_x86_WLK.exe"
+            os_dirs = Win2008 2k8
+        Win7:
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_x86_HCK.exe"
+            os_dirs = Win7 w7
+        Win8..0:
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_x86_HCK.exe"
+            os_dirs = Win8 w8
+        Win8..1:
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_x86_HCK.exe"
+            os_dirs = Win8.1 w8.1
+        Win10:
+            signtool = "WIN_UTILS:\signtool\signtool-HLK\Signtool_x86_HLK.exe"
+            os_dirs = Win10 w10
+    x86_64:
+        floppy_name = isos/windows/virtio-win.latest_prewhql.vfd.x86_64
+        arch_dir = 64
+        Win2008..sp2:
+            signtool = "WIN_UTILS:\signtool\signtool-WLK\Signtool_amd64_WLK.exe"
+            os_dirs = Win2008 2k8
+        Win2008..r2:
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe"
+            os_dirs = Win7 2k8R2
+        Win7:
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            os_dirs = Win7 w7
+        Win8..0:
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            os_dirs = Win8 w8
+        Win8..1:
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            os_dirs = Win8.1 w8.1
+        Win2012..r1:
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            os_dirs = Win8 2k12
+        Win2012..r2:
+            signtool = "WIN_UTILS:\signtool\signtool-HCK\Signtool_amd64_HCK.exe" 
+            os_dirs = Win8 2k12R2
+        Win10:
+            signtool = "WIN_UTILS:\signtool\signtool-HLK\Signtool_amd64_HLK.exe"
+            os_dirs = Win10 w10
+        Win2016:
+            signtool = "WIN_UTILS:\signtool\signtool-HLK\Signtool_amd64_HLK.exe"
+            os_dirs = Win10 2k16
```

Comment:
Maybe you can make signtool and guest name into a map,just as "guest_alias" in "single_driver_install.cfg" does.
So is the os_dirs.

Reply:
lijin, yes, I also want to realize that, thanks~